pkg/resource/unstructured/composite/composite.go

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
This should have been `git mv`-ed from `unstructured/composite.go`, sorry. :( I've become too trusting in git's ability to detect this automatically.